mmv1/products/vertexai/api.yaml

```diff
@@ -37,6 +37,9 @@ objects:
           'https://cloud.google.com/vertex-ai/docs'
       api: 'https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.datasets'
     async: !ruby/object:Api::OpAsync
+      actions:
```

Comment:
Looks like this fixes another bug that exists in dataset. https://github.com/hashicorp/terraform-provider-google/issues/12799